CHANGELOG.md

```diff
@@ -1,10 +1,24 @@
-Version 0.0.3
- - Change Python rules to srcs_version = "PY2AND3"
-
-Version 0.0.2
- - Add PhantomJS browser for Linux.
- - Add support for named files in archives.
- - Add MacOS support for Chrome, Firefox, and PhantomJS browser.
- - Support placeholders for named file paths in capabilities.
- - Use template files to generate test scripts.
- - Add Python Library for use with web_test.
+# Change Log
```

Comment:
I find it easier to create detailed release notes this way, and I have had users request CHANGELOGS.md in my other projects.